web/src/styles/base/global.scss

```diff
@@ -183,6 +183,10 @@ ol {
   width: 11rem;
 }
 
+.width-15rem {
+  width: 15rem;
+}
+
 .minh-30 {
   min-height: 30rem;
 }
```

Comment:
Less to do with your code, and more of an observation. In some places we're defining the unit at the end of the class name and in others we're not. This should be consistent.

Reply:
Yeah we do, I was trying to follow the pattern of the one above it which is `.width-11rem`, but to your point we should standardize them at some point. The only reason I can think of to keep the units is to differentiate between custom and USWDS values? Which for example already has a`width-15`css class.

Reply:
I'm in favor of keeping `-rem`. Slightly more verbose, but more explicit.